frameworks/kafka/universe/marathon.json.mustache

```diff
@@ -125,7 +125,7 @@
     "KAFKA_ZOOKEEPER_URI": "{{kafka.kafka_zookeeper_uri}}",
 
     {{#kafka.kafka_advertise_host_ip}}
-    "TASKCFG_ALL_KAFKA_ADVERTISE_HOST" : "set",
+    "TASKCFG_ALL_KAFKA_ADVERTISE_HOST" : "true",
```

Comment:
This change was not strictly required, correct?

Reply:
This is, setup helper is checking the ADVERTISE_HOST env variable with a parser that doesn't accept set as true.